packages/upward-spec/package.json

```diff
@@ -28,7 +28,7 @@
     "apollo-server": "~2.3.1",
     "chalk": "~2.4.2",
     "csv-parse": "~4.3.0",
-    "graphql": "~14.0.2",
+    "graphql": "~14.1.1",
```

Comment:
I had to bump these versions due to an "incompatibility" bug where multiple instances of the `graphql` dependency were installed and the plugin didn't know which one to use.

Right now these are manually kept in sync. I would very much welcome a better way to do this, it seems to me like the version of `graphql` in `upward-js` and `upward-spec` should be completely independent.

Reply:
Should this just be a peer dep of the consuming project? IMO the more peer deps we delegate to the final consumer, the better, since that's a great way to prevent duplicate packages in a final bundle.

Reply:
Do `upward-js` and `upward-spec` even depend on the `graphql` dependency? I see a single [require('graphql-tag') here](https://github.com/magento-research/pwa-studio/blob/93cee6854e18c98c5fd31b51473147f0621662d7/packages/upward-js/lib/compiledResources/GraphQLDocument.js#L2) but other than that I don't see anything.

I also have next to no familiarity with these codebases though 😛